keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -112,3 +131,5 @@ class ImageService(
     return true
   }
 }
+
+private fun AppVersion.toImageName() = "$packageName-$version-h$buildNumber.$commit"
```

Comment:
ay ay ay I wish there was a better way

Reply:
Yeah, I'm sure I'm missing some fuck-awful edge case, but I don't believe any of those fields are allowed to be `null`.